CONTRIBUTING.md

```diff
@@ -124,19 +124,16 @@ pnpm exec changeset
 We have benchmarks to keep performance under control. You can run these by running (from the project root):
 
 ```shell
-pnpm run benchmark --filter astro
+pnpm run benchmark
 ```
 
-Which will fail if the performance has regressed by **10%** or more.
-
-To update the times cd into the `packages/astro` folder and run the following:
+This will run all available benchmarks sequentially. To run a specific benchmark, you can run (for example):
```

Comment:
Just a shout out that it can sometimes be confusing to mix what happens for two different commands on the same line like this. 

Because of the deleted lines, I can't actually suggest here, but I would propose a structure closer to the following so you can scan for actions like "To do X..." or "Use Y" etc at the beginnings of lines:

``````
We have benchmarks to keep performance under control.**

You can run all available benchmarks sequentially by running (from the project root):

```shell
pnpm run benchmark
```

To run a specific benchmark only, you can add the name of the benchmark after the command:

```shell
pnpm run benchmark memory
```

Use `pnpm run benchmark --help` to see all available options.
``````

** (not SUPER required to start a new line here! Feel free to keep these sentences combined! But especially if you wanted to add any sentence with generic benchmark info, or if you still had that note about failure for example, this would be the place to do it. Then start each new line with an action someone might want to do, followed by the command.)

Reply:
I like the convention! I've split them up to follow the suggestion. I've also added more info of the `benchmark` directory, and the `!bench` command. Let me know what you think!